### PR TITLE
Got rid of lock

### PR DESCRIPTION
**Pull request by valdisiljuconoks**

### CODE REVIEW

File: src/Geta.GoogleProductFeed/NamespacedXmlMediaTypeFormatter.cs [hunk 1]
```diff
@@ -12,45 +12,30 @@ namespace Geta.GoogleProductFeed
...
-            {
-                if (!Serializers.ContainsKey(type))
-                {
-                    var serializer = new XmlSerializer(type, "http://www.w3.org/2005/Atom");
-
-                    Serializers.Add(type, serializer);
-                }
-            }
+            if(!_serializers.ContainsKey(type))
+                _serializers.TryAdd(type, new XmlSerializer(type, "http://www.w3.org/2005/Atom"));
```
Comment (frederikvig) [inline src/Geta.GoogleProductFeed/NamespacedXmlMediaTypeFormatter.cs]: Let's add space between if and ( and brackets around {}

File: src/Geta.GoogleProductFeed/NamespacedXmlMediaTypeFormatter.cs [hunk 2]
```diff
@@ -12,45 +12,30 @@ namespace Geta.GoogleProductFeed
...
-                var writerSettings = new XmlWriterSettings
-                {
-                    OmitXmlDeclaration = false
-                };
-
-                var xmlWriter = XmlWriter.Create(writeStream, writerSettings);
-                serializer.Serialize(xmlWriter, value, Namespaces);
-            });
+                                         {
+                                             if(!_serializers.TryGetValue(type, out var serializer))
```
Comment (frederikvig) [inline src/Geta.GoogleProductFeed/NamespacedXmlMediaTypeFormatter.cs]: Same as above

File: src/Geta.GoogleProductFeed/NamespacedXmlMediaTypeFormatter.cs [hunk 3]
```diff
@@ -12,45 +12,30 @@ namespace Geta.GoogleProductFeed
...
+                                                 return;
+
+                                             var writerSettings = new XmlWriterSettings { OmitXmlDeclaration = false };
```
Comment (frederikvig) [inline src/Geta.GoogleProductFeed/NamespacedXmlMediaTypeFormatter.cs]: Is the formatting off?

File: src/Geta.GoogleProductFeed/NamespacedXmlMediaTypeFormatter.cs (hunk 1 above)
Comment (valdisiljuconoks) [reply src/Geta.GoogleProductFeed/NamespacedXmlMediaTypeFormatter.cs]: fixed in reformat branch. will fix when merging

Comment (valdisiljuconoks) [reply src/Geta.GoogleProductFeed/NamespacedXmlMediaTypeFormatter.cs]: https://github.com/Geta/GoogleProductFeed/blob/master/src/Geta.GoogleProductFeed/Controllers/GoogleProductFeedController.cs#L25

Comment (valdisiljuconoks) [reply src/Geta.GoogleProductFeed/NamespacedXmlMediaTypeFormatter.cs]: we have to be consistent then everywhere :)

**Pull request merged**